autonomy/data/abis/registries_manager.json

```diff
@@ -0,0 +1,534 @@
+[
```

Comment:
I don't think we should ship these ABI files with the framework, this will increase the wheel size by a good amount. We can download an cache them in the `.aea` directory